app/components/logout.controller.js

```diff
@@ -1,12 +1,11 @@
 ;(function(){
   angular
     .module("todo")
+    .controller("LogoutController", LogoutController);
 
-    .controller("LogoutController", LogoutController)
-
-    LogoutController.$inject = ["$scope"]
+    LogoutController.$inject = ["$scope"];
 
     function LogoutController($scope){
-      $scope.user = null
+      $scope.user = null;
     }
-})()
+})();
```

Comment:
the same code
https://github.com/NikitaSedyx/todo-list/blob/master/app/components/logout.controller.js